lib/node_modules/@stdlib/stats/base/dcumax/src/main.c

```diff
@@ -30,26 +31,35 @@
 * @param Y        output array
 * @param strideY  Y stride length
 */
-void stdlib_strided_dcumax( const int64_t N, const double *X, const int64_t strideX, double *Y, const int64_t strideY ) {
+void API_SUFFIX( stdlib_strided_dcumax ) ( const CBLAS_INT N, const double *X, const CBLAS_INT strideX, double *Y, const CBLAS_INT strideY ) {
+	const CBLAS_INT ox = stdlib_strided_stride2offset( N, strideX );
+	const CBLAS_INT oy = stdlib_strided_stride2offset( N, strideY );
+	return API_SUFFIX(stdlib_strided_dcumax_ndarray)( N, X, strideX, ox, Y, strideY, oy );
+}
+
+/**
+* Computes the cumulative maximum of double-precision floating-point strided array elements using alternative indexing semantics.
+*
+* @param N        number of indexed elements
+* @param X        input array
+* @param strideX  X stride length
+* @param Y        output array
+* @param strideY  Y stride length
+* @param offsetX  starting index for X
```

Comment:
These parameter descriptions are in the wrong order and don't match the signature.